scipy/integrate/tests/test_cubature.py

```diff
@@ -24,18 +25,23 @@
 
 
 def basic_1d_integrand(x, n):
-    x_reshaped = x.reshape(-1, 1, 1)
-    n_reshaped = n.reshape(1, -1, 1)
+    xp = array_namespace(x, n)
```

Comment:
would it be possible to add an `xp` argument to these functions so that we don't need to use `array_namespace` when we already have `xp` in the test functions?